src/elfpy/strategies/LP_and_withdraw.py

```diff
@@ -0,0 +1,36 @@
+from elfpy.strategies.basic import BasicPolicy
+
+
+class Policy(BasicPolicy):
+    """
+    simple LP
+    only has one LP open at a time
+    """
+
+    def __init__(self, market, rng, wallet_address, budget=1000, amount_to_LP=100, verbose=False):
```

Comment:
Why is `amount_to_lp` included in the constructor? I think elsewhere we were setting user variables in the init itself, but not in the constructor.

Reply:
We were indeed doing that (e.g. look at single_short.py and single_long.py), but I feel like being able to set the trade amounts in the constructor makes individual tests a bit more flexible. Wdyt?

Reply:
Yeah I think that makes sense. I am worried that you shouldn't add variables to subclasses on __init__, it kind of defeats the point of the parent class. We use those to establish an API for downstream implementations, but if we're always overwriting the API then it is an anti-pattern. I think you'd be better to limit the number of named args to an essential subset, and then use `**kwargs` for the rest.

Reply:
leaving as is for now, will consider improving in future PR